### ISSUE TITLE
"Can't locate JSON.pm in @INC" error on createdb.sh in development setup

### ISSUE THREAD

**Issue by loujine** [metabrainz/musicbrainz-docker#161]
I'm trying to follow the doc on installing a [development setup](https://github.com/metabrainz/musicbrainz-docker#development-setup) for the first time and the createdb.sh command fails with the "Can't locate JSON.pm in @INC" error.

Looks it's fixed if I use "-Mlocal::lib -Mlocal::lib="${MUSICBRAINZ_PERL_LOCAL_LIB}"" to the perl eval command in the createdb.sh script.

Can somebody confirm?

**Comment by yvanzo** [comment]
Hi @loujine, I can see where this error is coming from: The Perl module `JSON` is installed by the script `start.sh` but the installation instructions say to run `createdb.sh` first. Might it be you started the container in the meantime? That would more likely have fixed your error.

Maybe we should split the init part off from `start.sh` and source it from `createdb.sh`.

**Comment by loujine** [comment]
@yvanzo : could be, I had trouble reproducing the problem without completely pruning my docker system. I just wanted to open an issue in case I was not the only one
Thanks

**Comment by yvanzo** [comment]
Thanks for reporting the issue and trying to address it, I will work on fixing scripts and doc.

**Comment by navap** [comment]
I can also confirm this issue when starting from scratch on Ubuntu 20.04 and performing the dev setup steps outlined here:
https://github.com/metabrainz/musicbrainz-docker/blob/554b1574983f912c00ec7b623d6f505181e3ebbe/README.md#development-setup

**Comment by jesus2099** [comment]
Same problem for me on Debian 10, where [I installed Docker, Docker Compose and then I followed the development MusicBrainz server install procedure](https://community.metabrainz.org/t/how-difficult-is-it-to-setup-a-local-musicbrainz-server/459658/20?u=jesus2099).
I get this JSON.pm error that apparently prevents proper database setup.

And, by the way, I don't have that `eval` command in my `/build/musicbrainz-dev/scripts/createdb.sh` script file at all.
Not in `/build/musicbrainz/scripts/createdb.sh` either, for what it's worth.
